packages/universal-router-sdk/src/SwapRouterPancake.ts

```diff
@@ -0,0 +1,64 @@
+import { TradeType } from '@pancakeswap/sdk'
+import { SmartRouter, SmartRouterTrade } from '@pancakeswap/smart-router/evm'
+import { MethodParameters } from '@pancakeswap/v3-sdk'
+import invariant from 'tiny-invariant'
+import { encodeFunctionData, toHex } from 'viem'
+import abi from './abis/UniversalRouter.json'
+import { PancakeSwapTrade } from './entities/protocols/pancakeswap'
+import { encodePermit } from './utils/inputTokens'
+import { RoutePlanner } from './utils/routerCommands'
+import { PancakeSwapOptions, SwapRouterConfig } from './utils/types'
+
+export abstract class PancakeUniversalSwapRouter {
```

Comment:
Let's rename to `PancakeswapUniversalRouter`

Reply:
yup renamed it